lib/log.js

```diff
@@ -560,12 +560,21 @@ const util = require('util'),
             }
 
             options.argv.argv.remain[0] = path.resolve(options.argv.argv.remain[0]);
-            if (fs.existsSync(options.argv.argv.remain[0])) {
-                fs.unlinkSync(options.argv.argv.remain[0]);
-            }
         }
-
         log.verbose("log#createLogFile()", "options.argv.argv.remain[0]:" + options.argv.argv.remain[0]);
+
+        // For error handling of non exist path
+        fs.open(options.argv.argv.remain[0], 'w', function(err, fd) {
```

Comment:
Why don't you use assign this value instead of using the remain[0]?

Reply:
I've update this.